Source/DafnyLanguageServer/DafnyLanguageServer.cs

```diff
@@ -11,9 +12,11 @@
 using OmniSharp.Extensions.LanguageServer.Server;
 using System.Threading;
 using System.Threading.Tasks;
+using Microsoft.Extensions.Options;
 
 namespace Microsoft.Dafny.LanguageServer {
   public static class DafnyLanguageServer {
+    public static IReadOnlyList<string> LoadErrors = new List<string>();
```

Comment:
Nitpicking as of now, but I'd like to avoid (mutable) static fields within the language server as much as possible.
However, we may want to consider introducing a global state object for the language server in the future for such cases.